From f6d56c4a0cceffa0a3def7e32a0f7c470e86d652 Mon Sep 17 00:00:00 2001
From: Christian Marangi <ansuelsmth@gmail.com>
Date: Mon, 18 Aug 2025 17:12:50 +0200
Subject: [PATCH] kernel: pf_ring: fix old-style-declaration compilation
 warning

New kernel linux from 6.12 enabled new compilation Warning and this
cause errors if -Werror is enabled.

There is currently one line that triggers the following warning:

pf_ring.c:284:1: error: 'static' is not at beginning of declaration [-Werror=old-style-declaration]
  284 | const static ip_addr ip_zero = { IN6ADDR_ANY_INIT };
      | ^~~~~

Implement the trivial fix to mute the warning.

Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
---
 kernel/pf_ring.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/kernel/pf_ring.c
+++ b/kernel/pf_ring.c
@@ -269,7 +269,7 @@ static inline void printk_addr(u_int8_t
 
 /* ************************************************* */
 
-const static ip_addr ip_zero = { IN6ADDR_ANY_INIT };
+static const ip_addr ip_zero = { IN6ADDR_ANY_INIT };
 
 static u_int8_t pfring_enabled = 1;
 
